### ISSUE TITLE
Integrate into "Filter by title"

### ISSUE THREAD

**Issue by NKarolak** [MicrosoftDocs/dynamics365smb-devitpro-pb#1906]
Please add this article to the "Filter by title" search.
![image](https://user-images.githubusercontent.com/34504100/121857266-b9ba9a00-ccf5-11eb-920d-5de4db55cd29.png)

Currently, it can be found only by "Search in all BusinessCentral documentation".

The article displayed in the screenshot is another one:
https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/properties/devenv-format-property

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 286fe842-d3a2-ba49-77d4-4513e2ce350a
* Version Independent ID: dfc559a5-1f4c-8a61-7b58-76f5d2771397
* Content: [Format Property - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-format-property)
* Content Source: [dev-itpro/developer/devenv-format-property.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/main/dev-itpro/developer/devenv-format-property.md)
* Service: **dynamics365-business-central**
* GitHub Login: @SusanneWindfeldPedersen
* Microsoft Alias: **solsen**

**Comment by SusanneWindfeldPedersen** [comment]
Hello, thanks for the feedback. We have an update on the way to include the mentioned topic in the TOC for searchability through Filter by title. A bit of background is that the topic isn't really a property topic and was moved out of that folder when we started auto-generating properties - hence the topic is updated and included in TOC at a different level with the next doc refresh. Hope that makes sense, Thanks!
